packages/eslint-plugin-next-on-pages/tests/rules/no-unsupported-configs.test.ts

```diff
@@ -123,7 +123,7 @@ describe('no-unsupported-configs', () => {
     /** @type {import('next').NextConfig} */
     const nextConfig = {
       assetPrefix: 'test',
-      reactStrictMode: true,
```

Comment:
What happened to `reactStrictMode`?

Reply:
This test is for the _currently_ unsupported configs, but I updated `reactStrictMode` to be _indefinitely_ unsupported because I don't think we're likely to ever support it since it's a development mode only feature ([docs](https://nextjs.org/docs/app/api-reference/next-config-js/reactStrictMode)) and we don't suggest the use of `next dev` (but rebuild the whole app altogether each time).

(this might change at some point if we decide to rely on `next dev` for development, but if we keep the current implementation I don't think we'll ever be able to enable this)